resources/js/components/UserEditForm.vue

```diff
@@ -245,9 +245,11 @@
 </template>
 
 <script>
-import {alpha, email, maxLength, minLength, required} from 'vuelidate/lib/validators';
+import { email, maxLength, minLength, required, helpers } from 'vuelidate/lib/validators';
 import notGTEmail from '../customValidators/notGTEmail';
 
+const alphaSpace = helpers.regex('alphaSpace', /^[a-zA-Z ]*$/);
```

Comment:
This allows there to be multiple spaces between words as well as leading and trailing spaces. I imagine we trim the input here so leading/trailing spaces aren't concerning, but can we adjust the regex to allow only one space between words?

Some testing suggests `^([a-zA-Z]+\s)*[a-zA-Z]+$` would work better (https://stackoverflow.com/a/21722204), but it's worth noting that either this or the existing proposal will block preferred names containing accents.  It's possible to write a [regex to allow accented character](https://stackoverflow.com/q/20690499)s, but not sure if we want/need to go down that rabbit hole.

Another alternative: write a looser regex to block numbers and most special characters, and replace all whitespace between words with just one space server-side

Reply:
I switched to your suggested regex. Blocking accents is deliberate. We blocked them before and I think expanding to allow them is out of scope for this PR. Additionally GT itself removes accents.

There was no server-side validation of this before. I added it.